### PR TITLE
Create table to store list of jdr reports for a middleware server

### PR DESCRIPTION
**Pull request by israel-hdez**
Jira: https://issues.jboss.org/browse/HAWKULAR-1231

### CODE REVIEW

File: db/migrate/20170720194715_create_middleware_jdr_reports.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+class CreateMiddlewareJdrReports < ActiveRecord::Migration[5.0]
+  def change
+    create_table :middleware_jdr_reports do |t|
```
Comment (Fryguy) [inline db/migrate/20170720194715_create_middleware_jdr_reports.rb]: jdr sounds very java specific.  If there were an alternate non-java middleware provider what would the equivalent/generic term be?

Comment (Fryguy) [inline db/migrate/20170720194715_create_middleware_jdr_reports.rb]: On that note, what *is*  JDR report?  I can't find anything googling.

Comment (israel-hdez) [inline db/migrate/20170720194715_create_middleware_jdr_reports.rb]: JDR = JBoss diagnostic Reporter.
I guess the generic term would be middleware_diagnostic_reports.

Comment (Fryguy) [inline db/migrate/20170720194715_create_middleware_jdr_reports.rb]: :+1: for middleware_diagnostic_reports

Comment (israel-hdez) [inline db/migrate/20170720194715_create_middleware_jdr_reports.rb]: BTW, do you expect a generic model in main repo? Right now, I only have a model in this PR: ManageIQ/manageiq-providers-hawkular#30

(May be you also know the answer to https://github.com/ManageIQ/manageiq-providers-hawkular/pull/30#issuecomment-319691445 on that same PR)

Comment (abonas) [inline db/migrate/20170720194715_create_middleware_jdr_reports.rb]: @jdoyleoss please address the above terminology discussion regarding JDR

**Comment by israel-hdez** [comment]
Besides the possibility of a rename, no additional changes. I'll un-wip it.
@miq-bot remove_label wip

File: db/migrate/20170720194715_create_middleware_jdr_reports.rb (hunk 1 above)
Comment (jdoyleoss) [inline db/migrate/20170720194715_create_middleware_jdr_reports.rb]: I don't care what you call the table if it does not drive what's presented in the UI. In the UI we want to call it JDR or JBoss Diagnostic Report.

Comment (israel-hdez) [inline db/migrate/20170720194715_create_middleware_jdr_reports.rb]: @Fryguy I renamed the table. Please, confirm if a model is needed in main repo. If so, I'll add columns to enable STI.

**Comment by israel-hdez** [comment]
@abonas @bronaghs @Fryguy This is ready for review/merge. 
I'll insist on my [last comment](https://github.com/ManageIQ/manageiq-schema/pull/45#discussion_r131274207): no more changes here unless you need a generic model in main repo. If you need it, I'll add columns to enable rail's STI.

**Comment by abonas** [comment]
I am not an expert on models hierarchy requirement, hence I defer to @Fryguy @bronaghs

**Comment by abonas** [comment]
@miq-bot add_label providers/hawkular

**Comment by abonas** [comment]
@Fryguy @bronaghs any comments/merge?

**Comment by abonas** [comment]
@miq-bot add_label providers/middleware

**Pull request merged**